### ISSUE TITLE
On M1, the 'hidtest' executable in the 'hidapi' package reports a dylib error when executed.

### ISSUE THREAD

**Issue by katherinefluxinc** [Homebrew/homebrew-core#131504]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
[19:08:57 pm] ~ 🐈 brew gist-logs hidapi
Error: No logs.

[19:09:11 pm] ~ 🐈 brew config
HOMEBREW_VERSION: 4.0.18
ORIGIN: https://github.com/Homebrew/brew
HEAD: b0dc84b117f54f6b18e0528b95477c54e409ebdd
Last commit: 8 days ago
Core tap JSON: 20 May 21:52 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.3wpn3amA8Q/org.xquartz:0
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4-arm64
CLT: 14.3.0.0.1.1679647830
Xcode: 14.3
Rosetta 2: false

[19:09:21 pm] ~ 🐈 brew doctor
Your system is ready to brew.
```

### Verification

- [X] My "`brew doctor` output" says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.
- [X] I searched for recent similar issues at https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue and found no duplicates.

### What were you trying to do (and why)?

After installing the 'hidapi' package, I executed the command 'hidtest' to see information about my system's HID devices.

### What happened (include all command output)?

I ran this command and received the following error message:

```
[19:12:49 pm] ~ 🐈 hidtest 
dyld[23845]: Library not loaded: @rpath/libhidapi.0.dylib
  Referenced from: <F8BCDF2B-7C6E-368A-98C3-63D3ACC92A6F> /opt/homebrew/Cellar/hidapi/0.13.1/bin/hidtest
  Reason: tried: '/tmp/hidapi-20230109-4328-1npfsrw/hidapi-hidapi-0.13.1/build/src/mac/libhidapi.0.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/tmp/hidapi-20230109-4328-1npfsrw/hidapi-hidapi-0.13.1/build/src/mac/libhidapi.0.dylib' (no such file), '/tmp/hidapi-20230109-4328-1npfsrw/hidapi-hidapi-0.13.1/build/src/mac/libhidapi.0.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/tmp/hidapi-20230109-4328-1npfsrw/hidapi-hidapi-0.13.1/build/src/mac/libhidapi.0.dylib' (no such file), '/usr/local/lib/libhidapi.0.dylib' (no such file), '/usr/lib/libhidapi.0.dylib' (no such file, not in dyld cache)
Abort trap: 6
```

### What did you expect to happen?

I expected 'hidtest' to print out information about my system's HID devices.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
1. Install the 'hidapi' package by executing the command 'brew install hidapi'.
2. Execute the 'hidtest' command and observe the error message.
```

**Comment by carlocab** [comment]
Fixed in #131652.

**Comment by katherinefluxinc** [comment]
The issue is indeed solved after running the given commands.

Thanks very much!
